### PR TITLE
Add basic react setup w/ deployment in github pages

### PR DESCRIPTION
**Pull request by nelsinofrancisco**
In this PR I'm pushing the standards to react project with deployment using GH-PAGES. 
Use `npm start` to check out the page in the browser or you can check out the [live version](https://nelsinofrancisco.github.io/math_magicians/). 

#  Implemented Requirements are listed below

## General requirements

- [x] Make sure that there are no linter errors.
- [x] Make sure that you used the correct Gitflow.
- [x] Make sure that you documented your work in a professional way.

## JavaScript requirements

- [x] Follow our list of best practices for JavaScript.

## Project requirements

- [x] The easiest way to create a React application is using Create React App (CRA) and following the instructions in this [guide](https://reactjs.org/docs/create-a-new-react-app.html#create-react-app).
- [x] The application should run in the browser without errors.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+  "version": "0.1.0",
+  "private": true,
+  "dependencies": {
+    "@testing-library/jest-dom": "^5.14.1",
+    "@testing-library/react": "^11.2.7",
+    "@testing-library/user-event": "^12.8.3",
+    "react": "^17.0.2",
+    "react-dom": "^17.0.2",
+    "react-scripts": "4.0.3",
+    "web-vitals": "^1.1.2"
```
Comment (dasileker) [inline package.json:13]: - [x] Please fix the dependencies  I cant see the landing page of the project 

klkl

**Review by dasileker — CHANGES_REQUESTED** [review]
### **Hi @nelsinofrancisco,** 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
-N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: package.json (hunk 1 above)
Comment (henatan99) [reply package.json:13]: This issue is resolved. Well done!! :clap: :clap: :clap:

**Review by henatan99 — APPROVED** [review]
Hi @nelsinofrancisco , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**